Makefile

```diff
@@ -124,26 +124,27 @@ cluster-api-provider-azure: $(CONTROLLER_GEN)
 api-docs: $(GENAPIDOCS)
 	hack/gen-api-docs.sh $(GENAPIDOCS) $(DIR)
 
+# generate the SelectorSyncSet template file to be applied via app-interface
 .PHONY: app-sre-saas-template
 app-sre-saas-template: hypershift
+	# do not set oidc-storage-provider-s3-* parameters as these are not used
 	bin/hypershift install \
-		--oidc-storage-provider-s3-bucket-name=bucket \
-		--oidc-storage-provider-s3-secret=oidc-s3-creds \
-		--oidc-storage-provider-s3-region=us-east-1 \
-		--oidc-storage-provider-s3-secret-key=credentials \
 		--platform-monitoring=None \
 		--enable-ci-debug-output=false \
 		--enable-admin-rbac-generation=true \
 		--private-platform=AWS \
-		--aws-private-region=eu-east-1 \
+		--aws-private-region-secret=aws-credentials \
+		--aws-private-region-secret-key=region \
 		--aws-private-secret=aws-credentials \
 		--aws-private-secret-key=credentials \
 		--external-dns-provider=aws \
 		--external-dns-secret=dns-credentials \
-		--external-dns-domain-filter=service.hypershift.example.org \
-		--external-dns-txt-owner-id=txt-owner-id \
+		--external-dns-domain-filter-secret=dns-credentials \
```

Comment:
I don't think there's a use case to be as flexible as requiring a configmap/secret per parameter.
To simplify all the business logic and also the UX, I think we should a contractual single configMap/secret with keys/values for all the allowed input.
So then when the configmap exists all input must come from there.
We can follow up on this later.

Reply:
We can definitely use the same secret for all, by specifying the same `-secret` values. But some systems actually sync secret-store data directly onto kube secrets. It may then become weird to et aws creds and dns config into the same secret?
Or do you mean "a single secret for all dns config" ?

We could also potentially default the `--external-dns-domain-filter-secret` value to the one of `--external-dns-secret`